src/base/memory_monitor_sr.cpp

```diff
@@ -618,6 +667,61 @@ namespace cubperf
   {
     m_aggregater.get_transaction_info (tran_count, info);
   }
+
+  void memory_monitor::stat_dump_debug ()
```

Comment:
It's not about heap module. Please separate it to another issue.

Reply:
I agree with him. I would be nice to handle another PR or issue.

Reply:
I devided it to another issue.